README.md

```diff
@@ -9,51 +18,133 @@ Badness  Function
     [...]
 ```
 
-"Badness" of a function is simply an approximation of how many unsafe expressions are evaluated during an evaluation of that function. For instance, marking unsafe functions with a `*`, suppose your function `f` calls functions `g*` and `h`. Furthermore, `h` calls `i*`. Then the badness of `f` is 2. Functions with high badness have a lot of opportunities to be memory unsafe.
+"Badness" of a function is simply an approximation of how many unsafe
+expressions are evaluated during an evaluation of that function. For instance,
+marking unsafe functions with a `*`, suppose your function `f` calls functions
+`g*` and `h`. Furthermore, `h` calls `i*`. Then the badness of `f` is 2.
+Functions with high badness have a lot of opportunities to be memory unsafe.
 
 ## Installation
 
-Make sure that you have the following requirements:
+Siderophile is [available via crates.io](https://crates.io/crates/siderophile),
+and can be installed with `cargo`:
+
+```console
+$ cargo install siderophile
```

Comment:
```suggestion
cargo install siderophile
```
`$` don't play nicely with GitHub's click-to-copy feature. So I recommend not including them.